docs/VerifStrat/source/quick_start.rst

```diff
@@ -33,15 +33,32 @@ You will need:
 
 #. A Linux machine (CORE-V-VERIF has been successfully run under Ubuntu, Debian and CentOS).
 #. Python3 and a set of plug-ins. The current plug-in list is kept in `$CORE_V_VERIF/bin/requirements.txt`.
-#. A GCC cross-compiler (aka "the `Toolchain <https://github.com/openhwgroup/core-v-verif/blob/master/mk/TOOLCHAIN.md#core-v-toolchain>`_"). Even if you already have a RISC-V toolchain, please do follow that link and read **TOOLCHAIN.md** for recommended ENV variables to point to it.
+#. A GCC cross-compiler (aka "the `Toolchain <https://github.com/openhwgroup/core-v-verif/blob/master/mk/TOOLCHAIN.md#core-v-toolchain>`_"). Even if you already have a RISC-V toolchain, please do follow that link and read `TOOLCHAIN.md <https://github.com/openhwgroup/core-v-verif/blob/master/mk/TOOLCHAIN.md>`_ for recommended ENV variables to point to it.
 #. `Verilator <https://veripool.org/guide/latest/install.html>`_.
 
-An easy way to get the Python plug-ins installed on your machine is::
+There are two ways to get the Python plug-ins installed. The first one, which installs them in the host machine, is::
 
    $ git clone https://github.com/openhwgroup/core-v-verif.git $CORE_V_VERIF
    $ cd $CORE_V_VERIF/bin
    $ pip3 install -r requirements.txt
 
+The second one, which is more complex and more robust, is to create a
```

Comment:
I caution about giving advice like this.  Everyone's compute environment will be different and we already get too many, "how do I setup my tools" questions.

Reply:
I agree with this. Although, the virtual python environments guarantee that we do not break other python projects by changing the versions of the python packages.

I will rephrase it, I will suggest it instead of looking that authoritative

Reply:
Sounds good.